code/Model/Submission/SubmittedFileField.php

```diff
@@ -87,53 +109,67 @@ public function getFormattedValue()
      */
     public function getExportValue()
     {
-        return ($link = $this->getLink()) ? $link : '';
+        return ($links = $this->getLinks()) ? implode("\r", $links) : '';
     }
 
     /**
      * Return the link for the file attached to this submitted form field.
      *
-     * @return string
+     * @return array|null
      */
-    public function getLink($grant = true)
+    public function getLinks($grant = true)
     {
-        if ($file = $this->getUploadedFileFromDraft()) {
-            if ($file->exists()) {
-                $url = $file->getURL($grant);
-                if ($url) {
-                    return Director::absoluteURL($url);
+        if ($files = $this->getUploadedFilesFromDraft()) {
+            return array_reduce($files, function ($links, $file) use ($grant) {
+                if ($file->exists()) {
+                    $url = $file->getURL($grant);
+                    if ($url) {
+                        $links[$file->ID] = Director::absoluteURL($url);
+                    }
                 }
-                return null;
-            }
+                return $links;
+            }, []);
         }
+        return null;
     }
 
     /**
      * As uploaded files are stored in draft by default, this retrieves the
-     * uploaded file from draft mode rather than using the current stage.
+     * uploaded files from draft mode rather than using the current stage.
      *
-     * @return File
+     * @return array<File>|null
      */
-    public function getUploadedFileFromDraft()
+    public function getUploadedFilesFromDraft()
```

Comment:
We can't just rename methods outside of a major release.
Either target this to the `7` branch or implement a new method and deprecate the old one.